code/OpenAuditTool.py

```diff
@@ -246,6 +242,27 @@ def __init__(self, num_winners=50, max_num_it=2500, sample_by_size=False):
 
         # *** election data (manifests, reported votes, and reported outcomes)
 
+        # *** Election planning
+
+        # When deciding how much to sample, the number of winners required from
+        # completion simulations to decide to decrease the sample size.
+        e.num_winners = num_winners
+
+        # The number of steps when doing a random walk, in deciding the sample size.
+        e.max_num_it = max_num_it
+
+        # Defaults to False. When set True, this means that we vary the sample
+        # size across counties, by doing a random walk.
+        e.sample_by_size = sample_by_size
+
+        # Defaults to False. When set True, this means that we vary the
+        # sample size across counties, by using discrete Robbins Monro
+        e.use_discrete_rm = False
+
+        # Defaults to None. Defines how we choose which county to sample next
+        # when contests span multiple counties.
```

Comment:
What might other valid values be, and how would they help? How are they chosen if this is None?
Might referring to the command-line help and augmenting it be better for these parameters?